internal/monitors/conviva/conviva.go

```diff
@@ -169,33 +168,35 @@ func (m *Monitor) fetchMetrics(contextTimeout time.Duration, semaphore chan stru
 				logger.Errorf("GET metric %s failed. %+v", metricConf.MetricParameter, err)
 				return
 			}
+			timestamp := time.Now()
 			dps := make([]*datapoint.Datapoint, 0)
 			for metricParameter, series := range res {
 				metricConf.logFilterStatuses(series.Meta.FiltersWarmup, series.Meta.FiltersNotExist, series.Meta.FiltersIncompleteData)
 				metricName := "conviva." + metricParameter
 				for filterID, metricValues := range series.FilterIDValuesMap {
+				L:
 					for i, metricValue := range metricValues {
 						select {
 						case <-ctx.Done():
 							logger.Error(ctx.Err())
 							return
 						default:
-							dp := sfxclient.GaugeF(
-								metricName,
-								map[string]string{"account": metricConf.Account, "filter": metricConf.filtersMap[filterID]},
-								metricValue)
-							// Checking the type of series and setting dimensions accordingly
+							dp := sfxclient.GaugeF(metricName, map[string]string{"account": metricConf.Account, "filter": metricConf.filtersMap[filterID]}, metricValue)
+							dps = append(dps, dp)
+							dp.Meta[dpmeta.NotHostSpecificMeta] = true
 							switch series.Type {
+							// Getting the latest time_series metric value and timestamp only
 							case "time_series":
-								dp.Timestamp = time.Unix(int64(0.001*series.Timestamps[i]), 0)
+								dp.Value = datapoint.NewFloatValue(metricValues[len(metricValues)-1])
+								// Series timestamps are in milliseconds
+								dp.Timestamp = time.Unix(series.Timestamps[len(series.Timestamps)-1]/1000, 0)
```

Comment:
Is there really no way to just request the latest value only? Check the docs closely.